lib/screens/explore_screens/manage_invites/interactor/usecases/load_invites_use_case.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:async/async.dart';
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/api/invite_repository.dart';
+import 'package:seeds/datasource/remote/api/profile_repository.dart';
+import 'package:seeds/datasource/remote/model/invite_model.dart';
+
+class GetInvitesUseCase {
+  final InviteRepository _inviteRepository = InviteRepository();
+  final ProfileRepository _profileRepository = ProfileRepository();
+
+  Future<InvitesDto?> run() async {
+    final account = settingsStorage.accountName;
+
+    final result = await _inviteRepository.getInvites(account);
+
+    if (result.isError) {
+      return null;
+    } else {
+      final List<InviteModel> invites = result.asValue!.value;
+      final Iterable<Future<Result>> futures = invites
+          .where((element) => element.isClaimed && element.account != null)
+          .map((e) => _profileRepository.getProfile(e.account!));
+
+      final List<Result> accounts = await Future.wait(futures);
+
+      return InvitesDto(accounts, invites);
+    }
+  }
+}
+
+class InvitesDto {
```

Comment:
What is a "Dto"?

Can we find a better name for this? Or maybe I am the only one not knowing what DTO stands for ;)

Reply:
Maybe it's ODT for Object Data Transfer

Reply:
Yeah its DTO my bad English again 🍎 
https://www.oscarblancarteblog.com/2018/11/30/data-transfer-object-dto-patron-diseno/

Reply:
Data Transfer Object